test/extensions/filters/http/local_ratelimit/BUILD

```diff
@@ -40,6 +40,9 @@ envoy_extension_cc_test(
     size = "large",
     srcs = ["local_ratelimit_integration_test.cc"],
     extension_names = ["envoy.filters.http.local_ratelimit"],
+    tags = [
+        "cpu:3",
```

Comment:
i experimentally added something similar here https://github.com/envoyproxy/envoy/pull/27243/files#diff-3ca343c3f15cbe6a9354d5c44647e85b92d118a4a01123b470c7866cc5c088adR62

i wasnt 100% clear how this works but iiuc its indicative to bazel - bazel schedules less local jobs according to their cpu reservation

i think in the RBE case it doesnt help - we get mostly 2-core workers (ive seen sometimes 16-core also but infrequently) - but i think this flag doesnt come into to play in that case

also ~related https://github.com/envoyproxy/envoy/pull/27243#issuecomment-1537373774

Reply:
It does help in RBE case as well. It will prevent scheduling two integration test processes from running on the same two core VM. Reduce CPU over subscription by half.